### ISSUE TITLE
update to cap 1.4

### ISSUE THREAD

**Issue by jaymccon** [aws-quickstart/quickstart-suse-cloud-application-platform#10]

**Comment by kevinayres** [comment]
Have to see why new installs are not selecting the latest from the suse chart repo.. As it stands now the installs are still using v1.3. Eirini can be enabled in 1.4 for tech preview only for select deployments.

**Comment by zoopster** [comment]
you are using the old charts here in the charts directory. I'd update if I knew how, but the uaa and scf charts.yaml ref the 1.3 versions.

**Comment by kevinayres** [comment]
Yup - thank ya sir. Louis is actually pulling the charts for 1.4 this week and testing and we'll work that into the QS shortly after. There was a reason for this at the time but now it's being updated.
